yarn.lock

```diff
@@ -2412,9 +2412,9 @@ flat-cache@^1.2.1:
     graceful-fs "^4.1.2"
     write "^0.2.1"
 
-flow-bin@^0.38.0:
-  version "0.38.0"
-  resolved "https://registry.yarnpkg.com/flow-bin/-/flow-bin-0.38.0.tgz#3ae096d401c969cc8b5798253fb82381e2d0237a"
+flow-bin@^0.42.0:
+  version "0.42.0"
+  resolved "https://unpm.uberinternal.com/flow-bin/-/flow-bin-0.42.0.tgz#05dd754b6b052de7b150f9210e2160746961e3cf"
```

Comment:
The build may be failing because of this URL. Did you mean to switch the domain from yarnpkg.com to uberinternal.com?